pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -534,6 +531,12 @@ stdenv.mkDerivation (finalAttrs: {
     (enableFeature withZimg "libzimg")
     (enableFeature withZlib "zlib")
     (enableFeature withZmq "libzmq")
+  # Sort into the options above if version is the oldest supported version
```

Comment:
Just put it into the correct position now.

Reply:
I did not do that because in a later pr I'm planning to add more to that list and this also makes it easier to sort

Reply:
Also sorting tools don't really work if they inserted in between regular enableFeature options

Reply:
That's not a huge issue IMHO. All new changes to this will be incremental. The intuitive thing to do here is to add new flags in the alphabetically correct place. I'll have an eye on that in reviews anyhow.

Reply:
done

Reply:
Now looking at the diff, the previous way did have the nice property of grouping version-dependant flags together. It's probably better this way long-term though.

Another idea I had on this was to make the enableFeature functions return lists with the arg in it to enable a tri-state (enable, disable, empty (default)) that can be chained with simple list concat.  
I'll experiment with that some more.